admin_guide/scheduling/descheduler.adoc

```diff
@@ -0,0 +1,305 @@
+
+[[admin-guide-descheduler-policies-anti]]
+=== Remove Pods Violating Inter-Pod Anti-Affinity
+
+The `RemovePodsViolatingInterPodAntiAffinity` strategy ensures that pods violating inter-pod anti-affinity are removed from nodes. 
+
+For example, *Node1* has *podA*, *podB*, and *podC*. *podB* and *podC* have anti-affinity rules that prohibit them from running on the same node as *podA*. *podA* will be evicted from the node so that *podB* and *podC* can run on that node. This situation could happen if the anti-affinity rule was applied when *podB* and *podC* were running on the node. 
+
+[source,yaml]
+----
+apiVersion: "descheduler/v1alpha1"
+kind: "DeschedulerPolicy"
+strategies:
+  "RemovePodsViolatingInterPodAntiAffinity": <1>
+     enabled: true
+----
+
+<1> Set this value to `enabled: true` to use this policy. Set to `false` to disable this policy.
+
+[[admin-guide-descheduler-policy]]
+== Create a configmap to store descheduler policy
+
+Create a xref:../../dev_guide.configmaps.adoc#dev-guide-configmaps[configuration map] for the descheduler in the `kube-system` project, so that it can be mounted as a volume inside a pod.
+
+----
+# oc create configmap descheduler-policy-configmap \
+     -n kube-system --from-file=<path-to-policy-dir/policy.yaml> <1>
+----
+
+<1> The path to the policy file you created.
+
+[[admin-guide-descheduler-job]]
+== Create the job specification
+
+Create a xref:../../dev_guide.configmaps.adoc#dev-guide-jobs[job configuration] for the descheduler and create the job.
+
+. Create the configuration file:
++
+[source,yaml]
+----
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: descheduler-job
+  namespace: kube-system
+spec:
+  parallelism: 1
+  completions: 1
+  template:
+    metadata:
+      name: descheduler-pod <1>
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: "true" <2>
+    spec:
+        containers:
+        - name: descheduler
+          image: descheduler
+          volumeMounts: <3>
+          - mountPath: /policy-dir
+            name: policy-volume
+          command:
+          - "/bin/sh"
+          - "-ec"
+          - |
+            /bin/descheduler --policy-config-file /policy-dir/policy.yaml <4>
+        restartPolicy: "Never"
+        serviceAccountName: descheduler-sa <5>
+        volumes:
+        - name: policy-volume
+          configMap:
+            name: descheduler-policy-configmap
+----
++
+<1> Specify a name for the job.
+<2> Configures the pod so that it will not be descheduled.
+<3> The volume name and mount path in the container where the job should be mounted.
+<4> Path to the xref:admin-guide-descheduler-policies[policy file] you created.
+<5> Specify the name of the service account you created.
++
+The policy policy-file is mounted as a volume from the config map.
+
+. Run the descheduler as a job in a pod:
++
+----
+# oc create -f descheduler-job.yaml
+----
++
+For example:
++
+----
+# oc create -f descheduler-job.yaml
+----
```

Comment:
Does this command make the descheduler "go"? Does it run once or keep going until you stop it?

Reply:
Yes Job will run descheduler in a pod.  Since it is going to run as Job, it will complete once descheduler exits. It wont run forever.